src/main/java/mate/academy/bookstore/service/impl/BookServiceImpl.java

```diff
@@ -45,6 +46,7 @@ public BookDto save(CreateBookRequestDto requestDto) {
     }
 
     @Override
+    @Transactional
```

Comment:
Just interesting, how `@Transactional` can be useful in get method?

Reply:
Not resolved